radix-transactions/src/validation/transaction_validator.rs

```diff
@@ -76,13 +81,16 @@ impl TransactionValidationConfig {
             max_subintent_depth: 0,
             min_tip_basis_points: 0,
             max_tip_basis_points: 0,
+            max_total_signer_signatures: usize::MAX,
+            max_total_references: usize::MAX,
         }
     }
 
     pub const fn cuttlefish() -> Self {
         Self {
+            max_references_per_intent: 512,
             v2_transactions_allowed: true,
-            max_subintent_count: 128,
+            max_subintent_count: 32,
```

Comment:
Ahh my bad - it looks like we already have `PreparationSettings.max_subintents_per_transaction` - (my fault for duplicating these) - perhaps these should be merged?

Or we should just read it off `validation_settings.preparation_settings.max_subintents_per_transaction` inside the validator if we want to check it again.

Reply:
Will remove `max_subintent_count`.